app/src/androidTest/java/com/coveros/coverosmobileapp/oauth/RestRequestInstrumentedTest.java

```diff
@@ -149,61 +178,94 @@ public void deliverError_withInvalidTokenError() throws JSONException {
     }
 
     @Test
-    public void parseNetworkResponse_withValidNetworkResponse() throws JSONException {
-        JSONObject dataJson = new JSONObject();
-        dataJson.put("format", "standard");
+    public void parseNetworkResponse_withValidNetworkResponse() {
+        JsonObject dataJson = new JsonObject();
+        dataJson.addProperty("format", "standard");
         byte[] dataBytes = dataJson.toString().getBytes();
         Map<String, String> headers = new HashMap<>();
 
-        Response<JSONObject> postRequestResponse = postRequest.parseNetworkResponse(new NetworkResponse(dataBytes, headers));
-        String actualPostFormat = postRequestResponse.result.getString("format");
+        Response<JsonObject> postRequestResponse = postRequest.parseNetworkResponse(new NetworkResponse(dataBytes, headers));
+        String actualPostFormat = postRequestResponse.result.get("format").getAsString();
 
         assertThat(actualPostFormat, equalTo("standard"));
 
-        Response<JSONObject> getRequestResponse = postRequest.parseNetworkResponse(new NetworkResponse(dataBytes, headers));
-        String actualGetFormat = getRequestResponse.result.getString("format");
+        Response<JsonObject> getRequestResponse = postRequest.parseNetworkResponse(new NetworkResponse(dataBytes, headers));
```

Comment:
Did you mean to call **getRequest**.parseNetworkResponse(...)?  (You are storing the return value in a variable named **get**RequestResponse.)